tox.ini

```diff
@@ -18,6 +18,8 @@ envlist =
     linkcheck,
     # Test with the latest PyMongo.
     test-pymongo-latest,
+    # Test with oldest supported PyMongo.
+    test-pymongo-oldest,
```

Comment:
I see we test with 4.9 and we test with latest. Do we have any tests (manual at least) for pymongo 4.10?

Reply:
Added tests for 4.10 and 4.11 stable.